aspnetcore/blazor/forms-validation.md

```diff
@@ -223,12 +230,383 @@ In the following example:
             ...
         }
     }
+}
+```
+
+> [!NOTE]
+> The framework API doesn't currently exist to clear manually added validation messages. Therefore, we don't generally recommend adding validation messages directly to a new <xref:Microsoft.AspNetCore.Components.Forms.ValidationMessageStore> for the form's <xref:Microsoft.AspNetCore.Components.Forms.EditContext>. The only way to clear manually added validation messages is to assign a new <xref:Microsoft.AspNetCore.Components.Forms.EditContext>, which isn't an ideal approach in most cases. Use a different approach for notifying users about business logic validation failures. Suitable approaches are covered in the following sections. For more information, see [How to clear the Editform validation messages with a button click in Blazor (dotnet/aspnetcore #24563)](https://github.com/dotnet/aspnetcore/issues/24563).
+
+## Business logic validation
+
+Business logic validation can be accomplished by using either or both of the following approaches:
+
+* [Display messages to the user](#display-messages-to-the-user)
+* [Use a custom business logic validation component](#use-a-custom-business-logic-validation-component)
+
+In the following examples:
+
+* The validation requires that the value of the ship description (`Description`) isn't `null` or empty if the user selects the `Defense` ship classification (`Classification`).
+* The placeholder `{SOLUTION NAMESPACE}` is the solution's namespace.
```

Comment:
I don't think this is something that's done elsewhere in the aspnet doc (it seems that using "BlazorSample" is more common)

Reply:
That's fair. I have spots going either way right now. There is inconsistency on it. I'll update that on the next pass. After all, when I get to working on the common sample app on https://github.com/dotnet/AspNetCore.Docs/issues/14216, that's exactly what will happen with many of these examples.